web/src/utils/requestHelpers.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+import jsonata from 'jsonata';
+import uuidv4 from 'uuid/v4';
+import { formDataToRequest, requestToFormData } from '../constants/form';
+
+/**
+ * Generate random string with fix length
+ * @param {number} length length of string
+ */
+export const randomRealmId = length => {
```

Comment:
is there a back end for this process? I feel like generating a realm id is better off done on the backend?

Reply:
not yet, missing the function to identify conflict names yet, will move to backend after that's in place.

Reply:
https://github.com/bcgov/realm-o-matic/issues/21